root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Scripts/touryo/common.js

```diff
@@ -603,81 +684,206 @@ function Fx_ShowMessageDialog(url) {
     url = url + "&Time=" + year_now + month_now + date_now + day_now + hour_now + minute_now + second_now;
 
     // マスクする。
-    Fx_MaskOn();
+    Fx_DialogMaskOn();
```

Comment:
@daisukenishino 

When users can scroll horizontally on parent screen, the width of mask is incorrect. So, user can operate the controls in the area that is not masked.

![image](https://cloud.githubusercontent.com/assets/9585990/25933582/bb815150-3652-11e7-9886-fb083872a556.png)